packages/devtools_extensions/example/README.md

```diff
@@ -0,0 +1,43 @@
+# Example DevTools Extension
+
+This is an end-to-end example of a DevTools extension, including
+- the extension Flutter web app that will be embedded in DevTools
+(`foo/packages/foo_devtools_extension`)
+- the parent package that provides the DevTools extension (`foo/packages/foo`)
+- the end-user application that depends on `package:foo` (`app_that_uses_foo`),
+which will trigger a load of the `pacakge:foo` DevTools extension when debugging
```

Comment:
How about ordering it differently:

This is an end-to-end example of a DevTools extension, including:
- the package that enables the DevTools extension for end-user applications (`foo/packages/foo`)
- the end-user application that depends on `package:foo` (`app_that_uses_foo`),
which will trigger a load of the `pacakge:foo` DevTools extension when debugging
this app with DevTools.
- the extension Flutter web app that will be embedded in DevTools
(`foo/packages/foo_devtools_extension`)

Reply:
re-ordered.